package.json

```diff
@@ -3,16 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "coveralls": "^3.0.2",
+    "axios": "^0.18.0",
     "classnames": "^2.2.6",
+    "coveralls": "^3.0.2",
     "dotenv-webpack": "^1.5.7",
+    "enzyme": "^3.6.0",
+    "enzyme-adapter-react-16": "^1.5.0",
+    "file-loader": "^2.0.0",
+    "jimp": "^0.5.3",
```

Comment:
@reiosantos why do we need `jimp` in social logins?

Reply:
I cannot remember the exact package that required this, but npm was complaining that `jimp` was not installed, so I had to install it. I do not personally require them, but `NPM` complains about them missing.

Reply:
I have removed `jimp` from `package.json`